WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -601,6 +615,11 @@ def my_competitions
                               .where(id: competition_ids.uniq)
                               .sort_by { |comp| comp.start_date || Date.today + 20.year }.reverse
     @past_competitions, @not_past_competitions = competitions.partition(&:is_probably_over?)
+    bookmarked_ids = current_user.competitions_bookmarked.pluck(:competition_id)
+    @bookmarked_competitions = Competition.includes(:delegate_report, :delegates)
+                                          .not_over
+                                          .where(id: bookmarked_ids.uniq)
+                                          .sort_by { |comp| comp.start_date || Date.today + 20.year }
```

Comment:
nit: Can we pull out `Date.today + 20.year` into a variable like `DISTANT_FUTURE` and reference that instead? (I think this is the second occurrence of it in this method)